// url_service.proto
syntax = "proto3";

package streamlit_server;

// 空のメッセージ型

// URL情報を含むメッセージ
message SendUrlRequest {
  string url = 1;
}
message SendUrlReply {}

service URLService {
  // 新しいURLを送信
  rpc SendUrl (SendUrlRequest) returns (SendUrlReply) {}
}

